modules/swagger-codegen/src/main/resources/python/configuration.mustache

```diff
@@ -22,10 +22,16 @@ class TypeWithDefault(type):
     def __call__(cls):
         if cls._default is None:
             cls._default = type.__call__(cls)
-        return copy.copy(cls._default)
+        return cls._two_level_copy(cls._default)
```

Comment:
did you try to use `copy.deepcopy(cls._default)` to see if that would provide the expected behavior?

Reply:
Yes, sure. Deepcopy fails with `TypeError: can't pickle thread.lock objects` cause we have reach attributes such as logger file/handler.